packages/terra-alert/src/Alert.jsx

```diff
@@ -89,19 +89,19 @@ const defaultProps = {
 const getAlertIcon = (type, customIcon) => {
   switch (type) {
     case AlertTypes.ALERT:
-      return (<span className={cx('icon')}><IconAlert /></span>);
+      return (<span className={cx('icon')}><IconAlert a11yLabel="Alert" /></span>);
```

Comment:
Do we need to localized this a11y label? I can't find `a11yLabel` in terra-icon so not sure how it is used.

Reply:
`a11yLabel` is a new prop that was added as part of the accessibility upgrade:
https://github.com/cerner/terra-core/blob/64b6b9fc41d741160887028e6b42155c51d40887/packages/terra-icon/src/IconBase.jsx

Reply:
Ah, it's in IconBase. Thanks for the reference.

So if this will create a `title` element with this label and be read by the screen reader, I wonder if it does need to be localized. Getting the translations for all of these can be tedious.

Reply:
updated to include translated messages for the terra-search-field and terra-alert  which are having translations messages already.
https://github.com/cerner/terra-core/pull/3640/commits/b6652a1e2420432b9585a211e15d4c5f59ab4c46 

For the other components updated the links to consume decorative icons. we might need to verify all the components that consumes terra-icon to find out the type of icon that should be used to make component accessible.